core/config_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2016-2019 Cristian Măgherușan-Stanciu
+// Licensed under the Open Software License version 3.0
+
+package autospotting
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
```

Comment:
I can remove this, but it makes writing assertions a lot easier. It also has a [package for testing binaries](https://godoc.org/gotest.tools/icmd), which I think will be useful to get `autospotting.go` under test.

Reply:
Thanks, I'll check it out.